R/scrape_val_env.R

```diff
@@ -0,0 +1,90 @@
+#' @title Retrieve validation environment
+#' @description Retrieves dependencies used in validation report.
+#' Includes: OS, R version, packages listed in the validation package DESCRIPTION (Depends/Imports/Suggests), packages
+#' present in current session.
+#' @param pkg path to package
+#' @return data.frame with columns:
+#'
+#' \itemize{
+#' \item \code{resource} "OS", "R" or package name
+#' \item \code{type} identifier of requirement type:
+#'     \itemize{
+#'         \item system - OS or R resource
+#'         \item package_req - from DESCRIPTION Depends/Imports of the package being validated
+#'         \item extended_req - from DESCRIPTION Suggests of the package being validated
+#'         \item session - packages in current workspace not captured via package_req/extended_req
+#'     }
+#' \item \code{detail} OS or version details
+#'
+#' }
+#' @importFrom here here
+#' @importFrom utils packageVersion sessionInfo
+#' @importFrom desc desc
+#' @export
+vt_scrape_val_env <- function(pkg = here()){
```

Comment:
would it make more sense to use `pkg = "."`? this is in alignment with devtools practices

Reply:
see `e5da851`